### PR TITLE
Year in review 2023

### PR DESCRIPTION
**Pull request by nikku**
This adds the year in review blog (2023 edition).

Closes https://github.com/bpmn-io/internal-docs/issues/872

### CODE REVIEW

File: src/blog/2024/003-2023-year-in-review.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+published_time: "2024-01-24T13:28:09+01:00"
+modified_time: "2024-01-24T13:28:09+01:00"
+
+releases:
+
+---
+
+
+<p class="introduction">
+  From custom form components to bpmn-js type declarations, between intelligent FEEL editing and core modeling UX improvements, 2023 was a busy, good year for the [bpmn.io project](https://bpmn.io/) and friends of our editing toolkits. Let us look back at what we accomplished and take a glimpse into the near future, too.
```
Comment (crobbins215) [inline src/blog/2024/003-2023-year-in-review.md:21]: "2023 was a busy, good year" doesn't sound right to me. Some suggestions to improve the flow

> From the introduction of custom form components to enhanced bpmn-js type declarations, and from advancements in intelligent FEEL editing to significant upgrades in core modeling UX, 2023 was indeed a busy and fruitful year for the [bpmn.io project](https://bpmn.io/) and our community. Let's take a moment to reflect on our achievements and glimpse into the near future.

File: src/blog/2024/003-2023-year-in-review.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+
+title: 2023 in Review
+layout: blogpost.hbs
+
+slug: 2023-year-in-review
+author: Nico Rehwaldt <https://github.com/nikku>
+description: "From custom form components to bpmn-js type declarations, between intelligent FEEL editing and core modeling UX improvements: This post recaps 2023, a great year for bpmn.io."
```
Comment (nikku) [inline src/blog/2024/003-2023-year-in-review.md:8]:
```suggestion
description: "From the introduction of custom form components to enhanced bpmn-js type declarations, and from advancements in intelligent FEEL editing to significant upgrades in core modeling UX: This post recaps 2023, a busy, good year for bpmn.io and the community."
```

File: src/blog/2024/003-2023-year-in-review.md (hunk 1 above)
Comment (nikku) [reply src/blog/2024/003-2023-year-in-review.md]: Thanks, updated!

**Comment by nikku** [comment]
Minor adjustment added as discussed with @barmac:

* We mention accessibility as a contribution.

**Pull request merged**